py/server/deephaven/experimental/s3_credentials.py

```diff
@@ -0,0 +1,92 @@
+#
```

Comment:
Missing module docs

Reply:
I added this file by mistake, the code has been moved to s3.py only.